init_docs/languages/English.txt

```diff
@@ -832,5 +833,8 @@
   "Local_Resources": "Local Resources",
   "Prompt_MeetUp_save":"Meetup saved! Do you want to invite?",
   "Pending_Resources": "Pending Resources",
-  "Confirm_Course":"Are you sure you want to delete this Course?"
+  "Confirm_Course":"Are you sure you want to delete this Course?",
+  "Has_Requested_Promote":  "has requested to be promoted for 'Manager' access.",
+  "Promote_Request_Rejected":  "Request to promote has been rejected.",
+  "Promote_Request_Accepted":  "Member has been successfully promoted."
 }
```

Comment:
Please add this new lines also to our other 3 languages `Spanish.txt`, `Urdu.txt` and specially `Arabic.txt`.

... sorry we can stop doing that only if crowdin is finally in place ;)